packages/calcite-components/src/components/menu-item/menu-item.e2e.ts

```diff
@@ -50,6 +50,39 @@ describe("calcite-menu-item", () => {
     expect(eventSpy).toHaveReceivedEventTimes(1);
   });
 
+  describe("href support", () => {
+    const testHref = "#nature";
+    const testEl = `<calcite-menu><calcite-menu-item id="Nature" text="Nature" href="${testHref}"></calcite-menu-item></calcite-menu>`;
+
+    it("should navigate to a new url when href provided and user interacts with click", async () => {
+      const page = await newE2EPage();
+      await page.setContent(html`${testEl}`);
+      const originalUrl = page.url();
+      await page.waitForChanges();
+
+      const menuItem = await page.find("calcite-menu-item");
+      await page.waitForChanges();
+      await menuItem.click();
+      await page.waitForChanges();
+      const newUrl = page.url();
+      expect(newUrl).toEqual(originalUrl + testHref);
```

Comment:
@driskull @jcfranco any other ideas for how to test this? All I could come up with is checking an for a named anchor with `#`, any `href` that changed the url, I wasn't having luck testing against.

Reply:
I think this works. I don’t think we can use Puppeteer’s `waitForNavigation` method, but this seems like a good alternative. If you were able to make the test fail when expected (e.g. removing/bypassing  the fix), this should be good to go. 🚀